src/posts/lucky_0.23.cr

```diff
@@ -0,0 +1,219 @@
+class Lucky023Release < BasePost
+  title "Lucky v0.23 is out, and it's huge!"
+  slug "lucky-0_23-release"
+
+  def published_at : Time
+    Time.utc(year: 2020, month: 6, day: 30)
+  end
+
+  def summary : String
+    <<-TEXT
+    Lucky v0.23 is out now. Lots of new features, bug fixes,
+    and even some performance boosts!
+    TEXT
```

Comment:
What do you think about listing some of the improvements so it stands out a bit more?

Maybe something like: "Improved memoization, more powerful browser tests, routing speed improvements, and more!"